app/src/componentLibrary/fieldTextFlex/fieldTextFlex.scss

```diff
@@ -36,6 +36,10 @@
   color: $COLOR--almost-black;
   resize: vertical;
 
+  &::-webkit-resizer {
+    background-image: url('./img/resizer.svg');
+  }
```

Comment:
This property doesn't support by firefox discus it with UX
https://caniuse.com/mdn-css_selectors_-webkit-resizer